app/logging.py

```diff
@@ -1,11 +1,14 @@
 import logging
+from pathlib import Path
 
 from .config import config, get_logs_dir
 
 
 def configure_logging():
+    log_path = Path(get_logs_dir(config) / "server.log")

```

Comment:
`get_logs_dir()` returns a `Path`, so you don't need the conversion here.

Reply:
Ok.